config/locales/en.yml

```diff
@@ -332,6 +332,11 @@ en:
   settings:
     edit:
       title: Edit settings
+      webauthn_credentials: Credentials
+      no_webauthn_credentials: You don't have any credentials
+      add_webauthn_credential: Add credential
+      webauthn_credential_note: A credential can be a security key, a fingerprint (e.g. Touch ID), etc.
```

Comment:
there is also Windows Hello but might confuse users more than necessary, not sure, maybe I should add it